dbt/adapters/databricks/impl.py

```diff
@@ -101,9 +101,19 @@ class DatabricksConfig(AdapterConfig):
     buckets: Optional[int] = None
     options: Optional[Dict[str, str]] = None
     merge_update_columns: Optional[str] = None
+    merge_exclude_columns: Optional[str] = None
     databricks_tags: Optional[Dict[str, str]] = None
     tblproperties: Optional[Dict[str, str]] = None
     zorder: Optional[Union[List[str], str]] = None
+    skip_non_matched_step: Optional[bool] = None
+    skip_matched_step: Optional[bool] = None
+    matched_condition: Optional[str] = None
+    not_matched_condition: Optional[str] = None
+    not_matched_by_source_action: Optional[str] = None
+    not_matched_by_source_condition: Optional[str] = None
+    target_alias: Optional[str] = None
+    source_alias: Optional[str] = None
+    merge_with_schema_evolution: Optional[bool] = None
```

Comment:
The way python models work, I would expect these to apply there too, as python models are created by first making a table in python from a DataFrame, and then running SQL against that table to do things like incremental merges.

Reply:
Hm... should I add some python model test? I have no idea if it will work now.

Reply:
I think it would be good to add a python test.  Skip for databricks_uc_sql_endpoint and databricks_cluster.  It should just work, because the merge is between SQL tables, even when using a python model.

Reply:
Ok, I'll look into it